solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -749,12 +779,11 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         // get the total supply
         uint256 totalSupply = poolToken.totalSupply();
 
-        // sync the balances to ensure no mismatch
-        syncReserveBalances(msg.value);
-
         uint256[2] memory oldReserveBalances;
         uint256[2] memory newReserveBalances;
-        (oldReserveBalances[0], oldReserveBalances[1]) = reserveBalances();
+
+        // transfer the fees and sync the balances to ensure no mismatch
+        (oldReserveBalances[0], oldReserveBalances[1]) = processNetworkFees(msg.value);
```

Comment:
old -> prev?

Reply:
This will changed code which is so far untouched in this PR. Are you sure you want to proceed?

Reply:
done